### ISSUE TITLE
Card not loading  the playlists 

### ISSUE THREAD

**Issue by r11a** [custom-cards/spotify-card#170]
Hello,

Since updated to version 2021.12.0b the card doesn’t load the playlists

works fine with version 2021.11.5

HA Version 2021.12.0b6
Thank you

**Comment by Efekent** [comment]
Just updated to non beta release 2021.12.0 same issue as @r11a 

Card does not load the playlists and it is not possible to choose the devices, I can't see anything wrong in the logs. All 5 devices are available in Home Assistant.
If I add the Spotify player entity in Lovelace as media control card, I can open up it's Media Library and browse through my lists.

Here how it shows at the moment:
![spot_cust_card](https://user-images.githubusercontent.com/41875369/145694754-a48c1191-3b77-4eef-b70c-b788c8c72f8d.PNG)

The card also did work for me at 2021.11.5

**Comment by hmmbob** [comment]
Some changes in Chromecast in 2021.12.0, probably fixed in https://github.com/fondberg/spotcast/pull/271

**Comment by Efekent** [comment]
Thanks @hmmbob just added the fixes you provide manually and everything is working again.

**Comment by Urchin73** [comment]
Thanks for the fix... It worked again for one day or so and now... same issue...
I'm on 2021.12.3
Any idea ?
Thanks in advance !

**Comment by Efekent** [comment]
> 

I just upgraded the core to 2021.12.3 and card is still working, There was a update of the Spotcast integration itself couple days ago, did you update it? Not the card but the integration, here is the update information:  https://github.com/fondberg/spotcast/releases/tag/v3.6.23

**Comment by Urchin73** [comment]
Yes, indeed, i installed the last update of spotcast 3.6.23. It worked well after the update ...  But the card is not loading now (since a day or two after the working update) I didn't change anything though... Running 5 accounts... none of them is loading.
Am I the only one ?

**Comment by gmcmicken** [comment]
Can you share your card config?

**Comment by peceq** [comment]
the same with my card, my config:
//
type: custom:spotify-card
account: default
country_code: PL
display_style: grid
grid_covers_per_row: '5'
limit: 40
known_connect_devices: []
grid_show_title: true
playlist_type: default
always_play_random_song: true
spotify_entity: media_player.spotify_lukasz
//

**Comment by rindlerblabla** [comment]
~~Same here. Am using spotcast v3.6.24, HA 2021.12.3, spotify-card v2.4.0. Just using a basic config.~~

Sorry, obviously my `sp_dc` and `sp_key` got updated somehow and wasn't valid anymore. Working again now.

**Comment by hdejongh** [comment]
i have the same problem. Card keeps saying "loading" while the MiniPlayer (type: custom:mini-media-player) works fine.

**Comment by ebotty** [comment]
> Card keeps saying "loading" while the MiniPlayer (type: custom:mini-media-player) works fine.

Same issue here, but also no devices showing up. My MiniPlayer is working too--able to see devices and playlists.

**Comment by rvandet** [comment]
Not all options that are in the Lovelace card, are described on the Github page.
I guess the documentation is lagging behind. Nothing new so far, but would be nice to see all options here.

**Comment by rvandet** [comment]
Would be nice if we could sort the playlists (alphabetic, date added, ...)
